source/lib/predicates/object.ts

```diff
@@ -129,4 +132,28 @@ export class ObjectPredicate extends Predicate<object> {
 			validator: (object: any) => keys.some(key => dotProp.has(object, key))
 		});
 	}
+
+	/**
+	 * Test an object to match the `shape` partially.
+	 *
```

Comment:
I think we need to elaborate on what "shape" means. Also what "partially" means; that it will ignore unexpected properties. Would be good with a quick doc example too, since this one isn't as clear as the other assertions. Same with `exactShape`.

Reply:
It should also mention that the shape comparison is deep.